proto/store/store/audit_log.proto

```diff
@@ -45,6 +45,16 @@ message AuditLog {
   // Metadata about the operation.
   RequestMetadata request_metadata = 11;
 
+  // Instance identifier for this Bytebase server instance
+  // Format: {source}-{timestamp}-{random}
+  // Used by log aggregators for deduplication and gap detection
+  string instance_id = 12;
```

Comment:
The name `instance_id` can be confusing since there is another `instance`.
Maybe we can call it `deploy_id`? There is a `DeployID` in profile.go.